clang/docs/checkedc/Bounds-Checking-for-LValue-Expressions.md

```diff
@@ -252,3 +256,21 @@ implemented before the April release of the Checked C compiler.
   likely than `MemberExpr *` to have unknown target bounds.
 - Priority 3: `ImplicitCastExpr *` and `CHKCBindTemporaryExpr *`. These kinds
   are less common in example code.
+
+## Lexicographic Ordering for PreorderASTs
+
+Lexicographic ordering for PreorderASTs leverages the ordering implemented in
+[CanonBounds.cpp](https://github.com/microsoft/checkedc-clang/blob/master/clang/lib/AST/CanonBounds.cpp).
+particularly the CompareExpr method.
+
+For two leaf nodes `N1` and `N2` of a PreorderAST, the result of
```

Comment:
It will be great if you can add two sentences before this sentence: 1) one for how two non-leaf nodes or operator nodes are ordered (based on the lexicographic ordering of the operator codes), and 2) another one for how an operator node and a leaf node are lexicographically ordered (operator node > leaf node).